added alternate ruler for communist germany

ItsFlor opened the pull request:
If germany doesnt ally the USSR when going communist, Heinrich Brandler can take over the party and become the new country leader. germany gets a new colour and name.

added portrait, slightly buffed the soviet alliance to compensate for the leader buffs

ItsFlor committed:
stuff

marijn211 commented:
Personally dont like the idea of replacing a leader you just fought a civil war for (also biased cuz Thalmann gets the poggers uniform change)

Also events should stick to the a/b/c standard

ItsFlor committed:
Merge branch 'Beta-Edits' into flor-communist-germany

ItsFlor committed:
finishing touches

ilkober reviewed (dismissed):
An event should provide options and not just forcefully change your leader.

Also, what is the colour change for?

ilkober commented on common/characters/GER.txt:
Character name should be stored in Characters file

ItsFlor replied:
Other option is the mutually exclusive focus for leader
colour change is like trotzkyist ussr, they get a different colour too

ilkober replied:
Event should make you be able to select a leader, not just forcing the change. Especially in context of potential USSR civil war and alignment of that kind

ItsFlor replied:
added an additional option now

ItsFlor committed:
Merge branch 'Beta-Edits' into flor-communist-germany

ItsFlor committed:
cleanup and small fixes

ItsFlor committed:
updated basically everything

ItsFlor committed:
Merge branch 'Beta-Edits' into flor-communist-germany

ItsFlor commented:
-updated tooltips for several focuses, making them more readable
-fixed issue with treaty of versailles path not being able to do the last focus
-added new name for communist germany, changes with alliance with the ussr or with brandler
-updated some focus icons to look more communist

ItsFlor committed:
some more stuff, i llost track of what exactly i did

ItsFlor committed:
this tooltip was ass

ItsFlor committed:
flags and renaming decision

ItsFlor committed:
flag changes, deleted old code

ItsFlor committed:
final

ItsFlor committed:
post-final

marijn211 committed:
Merge branch 'Beta-Edits' into flor-communist-germany

marijn211 reviewed (dismissed):
Error log

ItsFlor committed:
fixed small bug

marijn211 committed:
Merge branch 'Beta-Edits' into flor-communist-germany

SwedishNorwegian committed:
This isn't needed any more as it is baked into baseline techs

SwedishNorwegian committed:
Added TT to clear up Heinrich Brandler

SwedishNorwegian committed:
Nerfed Heinrich Brandler before merge

SwedishNorwegian reviewed (approved):
LGTM
